libs/shared/ui-components/src/SEO/SEO.tsx

```diff
@@ -1,14 +1,63 @@
 import { FC } from 'react';
 
 import Head from 'next/head';
+import { useRouter } from 'next/router';
 
 import { DomainVariant } from '@quansight/shared/types';
 
 import { TSEOProps } from './types';
 
-export const SEO: FC<TSEOProps> = ({ title, description, variant }) => (
-  <Head>
-    <title>{`${title} | ${DomainVariant[variant]}`}</title>
-    <meta name="description" content={description} />
-  </Head>
-);
+export const SEO: FC<TSEOProps> = ({ title, description, variant }) => {
```

Comment:
@Surafel-Getachew -- While piggybacking this onto the `SEO` component is clearly working, this social preview metadata isn't really part of the SEO of the pages.

Can you re-factor the social media `<meta>` logic into a new component... `SocialMeta`, say?

That would make for a better semantic separation.

Reply:
@bskinn, The reason I add it in SEO component is one it's the only component that is shared by all the pages with the information we need for example (title, description, domainVariant).
If we create `SocialMeta` component we have the option
1. To add `SocialMeta` component on every page

Reply:
Then a deeper refactor may be called for... where the component currently called `SEO` gets renamed to something like `HeadMeta`, and the `HeadMeta` includes both `SEO` and `SocialMeta`.

Reply:
@bskinn , actually we are going to do that refactor sooner or later because in the future we will not be using Quansight website logo and labs logo for each page. they may have their own images and in that case having a component that accepts the specific image of that page will be needed so it's just a matter of time so I've already implemented it.